docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hcls-cluster
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hcls-01
+  region: us-central1
+  zone: us-central1-c
+
+  network_name: hcls-cluster-net
+  subnetwork_name: primary-subnet
+  bucket_name_software: user-provided-software
+  bucket_name_inputs: input-bucket
+  bucket_name_outputs: output-bucket
+  use_deployment_name_in_bucket_name: false
```

Comment:
It is very tempting to set this to true just so you reduce the amount of name clashes you will have.  However, I haven't actually tried it so maybe the result is awkward

Reply:
This would work much better with IGC. Right now each group has to set up pre-existing-network-storage based on the bucket name. So it is important that it is defined globally and is static. If you include the deployment name then we also have to include it later when we look it up.

Reply:
makes sense. Thanks!